NitroxServer/Serialization/ServerProtobufSerializer.cs

```diff
@@ -9,14 +9,11 @@ namespace NitroxServer.Serialization
 {
     public class ServerProtobufSerializer
     {
-        private readonly RuntimeTypeModel model;
 
-        protected RuntimeTypeModel Model { get { return model; } }
+        protected RuntimeTypeModel Model { get; } = TypeModel.Create();
```

Comment:
Has to be protected because it's used by SubnauticaServerProtobufSerializer. Want me to change it to new protected method to add to the `model` so that this can be private?

Reply:
k we'll leave it